tests/tacacs/test_ro_user.py

```diff
@@ -20,6 +20,36 @@ def test_ro_user(localhost, duthosts, rand_one_dut_hostname, creds, test_tacacs)
         if fds[0] == "test":
             assert fds[4] == "remote_user"
 
+def test_ro_user_docker_ps(localhost, duthosts, rand_one_dut_hostname, creds, test_tacacs):
+    duthost = duthosts[rand_one_dut_hostname]
+
+    # Run as readonly use the command `sudo docker ps`
+    dutip = duthost.host.options['inventory_manager'].get_host(duthost.hostname).vars['ansible_host']
+    res = localhost.shell("sshpass -p {} ssh "\
+                          "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null "\
+                          "{}@{} sudo docker ps".format(
+            creds['tacacs_ro_user_passwd'], creds['tacacs_ro_user'], dutip))
+
+    # Verify that the command is allowed
+    stdout = res['stdout_lines']
```

Comment:
can you generalize the check. just check the return value, if it non-zero or non-zero.

I'd like you to cover all command defined in the sudoer file in this pr as well.

Reply:
can you address  this?